addon/components/object-list-view.js

```diff
@@ -31,6 +31,18 @@ export default FlexberryBaseComponent.extend(
   */
   _modelProjection: null,
 
+  /**
+    Flag indicates whether to look for changes of model (and displaying corresponding changes on control) or not.
+
+    If flag enabled component compares current detail array with used on component,
+    removes deleted and marked as deleted on model level records, adds created on model level records.
+
+    @property searchForContentChange
+    @type Boolean
+    @default false
+  */
+  searchForContentChange: false,
```

Comment:
Then object-list-view designed badly if so.
Ok. If you insist that setting is necessary, then it's value should be checked not inside anyway attached observer.
Observer should be attached only if setting value is 'true'.